build/Analyzers.ruleset

```diff
@@ -1,61 +1,33 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="Autofac Analyzer Rules" Description="Analyzer rules for Autofac assemblies." ToolsVersion="14.0">
-  <!-- https://github.com/dotnet/roslyn/blob/master/docs/compilers/Rule%20Set%20Format.md -->
+<RuleSet Name="Autofac Analyzer Rules" Description="Analyzer rules for Autofac assemblies." ToolsVersion="16.0">
```

Comment:
I think the comments about the rules were helpful. Can we revert this?

Reply:
(Keep the updated ToolsVersion, but put the comments back, I mean.)

Reply:
Yeah, I can do that; Visual Studio must have borked them.